api/api_ethereum.go

```diff
@@ -1098,25 +1096,29 @@ func (args *EthTransactionArgs) setDefaults(ctx context.Context, b Backend) erro
 	if args.MaxPriorityFeePerGas == nil || args.MaxFeePerGas == nil {
 		if b.ChainConfig().IsEthTxTypeForkEnabled(head.Number) && args.GasPrice == nil {
 			if args.MaxPriorityFeePerGas == nil {
-				// TODO-Klaytn: Original logic of Ethereum uses b.SuggestTipCap which suggests TipCap, not a GasPrice.
-				// But Klaytn currently uses fixed unit price determined by Governance, so using b.SuggestPrice
-				// is fine as now.
-				args.MaxPriorityFeePerGas = (*hexutil.Big)(fixedGasPrice)
+				args.MaxPriorityFeePerGas = (*hexutil.Big)(gasPrice)
 			}
 			if args.MaxFeePerGas == nil {
 				// TODO-Klaytn: Calculating formula of gasFeeCap is same with Ethereum except for
 				// using fixedBaseFee which means gasFeeCap is always same with args.MaxPriorityFeePerGas as now.
+				baseFee := fixedBaseFee
+				if isKIP71 {
+					baseFee = gasPrice
+				}
 				gasFeeCap := new(big.Int).Add(
 					(*big.Int)(args.MaxPriorityFeePerGas),
-					new(big.Int).Mul(fixedBaseFee, big.NewInt(2)),
+					new(big.Int).Mul(baseFee, big.NewInt(2)),
 				)
 				args.MaxFeePerGas = (*hexutil.Big)(gasFeeCap)
 			}
-			if args.MaxPriorityFeePerGas.ToInt().Cmp(fixedGasPrice) != 0 || args.MaxFeePerGas.ToInt().Cmp(fixedGasPrice) != 0 {
-				return fmt.Errorf("only %s is allowed to be used as maxFeePerGas and maxPriorityPerGas", fixedGasPrice.Text(16))
-			}
-			if args.MaxFeePerGas.ToInt().Cmp(args.MaxPriorityFeePerGas.ToInt()) < 0 {
```

Comment:
I think it could be normal check logic regardless of isKIP71 value.
MaxFeePerGas should not be smaller than MaxPriorityFeePerGas